modules/mono/csharp_script.cpp

```diff
@@ -2100,6 +2104,43 @@ void GD_CLR_STDCALL CSharpScript::_add_property_default_values_callback(CSharpSc
 		p_script->exported_members_defval_cache[name] = value;
 	}
 }
+
+void CSharpScript::get_docs(Ref<CSharpScript> p_script) {
+	Dictionary class_doc_dict;
+	class_doc_dict.~Dictionary();
+
+	GDMonoCache::managed_callbacks.ScriptManagerBridge_GetDocs(p_script.ptr(), &class_doc_dict);
+
+	p_script->docs.clear();
+
+	if (class_doc_dict.is_empty()) {
+		// Script has no docs.
+		return;
+	}
+
+	DocData::ClassDoc class_doc = DocData::ClassDoc();
```

Comment:
Add this to set the `DocData` inheritance chain properly, so the C# scripts show in the right place in the _Search Help_ dialog.

```suggestion
	String inherits;
	Ref<CSharpScript> base = p_script->get_base_script();
	if (base.is_null()) {
		// Must be a native base type.
		inherits = p_script->get_instance_base_type();
	} else {
		inherits = base->get_global_name();

		if (inherits == StringName()) {
			inherits = base->get_path();
		}
	}

    DocData::ClassDoc class_doc = DocData::ClassDoc();
    class_doc.inherits = inherits;
```

_This code was adapted from `create_dialog.cpp`._